Pipfile

```diff
@@ -4,35 +4,19 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-aiohttp = "==3.7.4.post0"
-async-timeout = "==3.0.1"
-attrs = "==20.3.0"
-cachetools = "==4.2.1"
-certifi = "==2020.12.5"
-chardet = "==4.0.0"
-discord = "==1.0.1"
-"discord.py" = "==1.6.0"
-google-auth = "==1.27.1"
-google-auth-oauthlib = "==0.4.3"
-gspread = "==3.7.0"
-gspread-formatting = "==0.3.7"
-idna = "==2.10"
-multidict = "==5.1.0"
-oauthlib = "==3.1.0"
-pyasn1 = "==0.4.8"
-pyasn1-modules = "==0.2.8"
-python-decouple = "==3.4"
-requests = "==2.25.1"
-requests-oauthlib = "==1.3.0"
-rsa = "==4.7.2"
-six = "==1.15.0"
-typing-extensions = "==3.7.4.3"
-urllib3 = "==1.26.3"
-yarl = "==1.6.3"
+"discord.py" = "*"
+python-decouple = "*"
 selenium = "*"
 flake8 = "*"
+gspread = "*"
+gspread-formatting = "*"
 
 [dev-packages]
+flake8 = "*"
+black = "*"
 
 [requires]
-python_version = "3.7"
+python_version = "3.10"
```

Comment:
cleanup on the pipe file. this is much sleeker and cleaner, and doesn't look like a requirements.txt file --> pipfile.